uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestRights.php

```diff
@@ -58,8 +58,9 @@ public function canChangeDatesFor($contactID, $statusID, $requestType) {
     $isSicknessRequest = $requestType === LeaveRequest::REQUEST_TYPE_SICKNESS;
     $isOpenLeaveRequest = in_array($statusID, $openStatuses);
 
-    $currentUserCanChangeDates = ($isSicknessRequest && $this->currentUserIsManagerOrAdmin($contactID)) ||
-                                 ($this->currentUserIsLeaveContact($contactID) && $isOpenLeaveRequest);
+    $currentUserCanChangeDates = ($isSicknessRequest && $this->currentUserIsManager($contactID)) ||
```

Comment:
Since this logic has been changed, we need to add/confirm the following tests:
(1) Tests that shows that an admin can now edit dates for a TOIL and Leave Request types .
(2) Tests that shows that a Leave Manager can only edit the dates of a Sickness Request and not that of TOIL/Leave (the tests should be there already).